esp/src/src-react/components/Menu.tsx

```diff
@@ -218,10 +218,6 @@ export const SubNavigation: React.FunctionComponent<SubNavigationProps> = ({
         return subNavSelectedKey(hashPath);
     }, [hashPath]);
 
-    const altSubNav = React.useMemo(() => {
```

Comment:
This function seemed to only be used for the "#/topology" left nav item. That's the only top level of the menu where there isn't a submenu that matches the root route, ie, there isn't actually a "#/topology" page.